libcodechecker/analyze/analysis_manager.py

```diff
@@ -190,6 +190,8 @@ def check(check_data):
     skipped = False
     reanalyzed = False
     try:
+        os.chdir(action.directory)
```

Comment:
instead of this, maybe we should specify the cwd parameter of the SourceAnalyzer.run_proc(analyzer_cmd,
                                                               env, cwd)
in analyzer_base.py.
I guess that would be cleaner.